src/main/java/com/bp3/camunda/camunda7/C7RestConnector.java

```diff
@@ -48,6 +49,18 @@ public final class C7RestConnector implements ExternalTaskHandler {
     private final HttpConnector httpConnector;
     private final ObjectMapper mapper;
 
+    @Value("${camunda.bpm.client.base-url}")
+    private String engineEndpoint;
+
+    @Value("${REST_TOPIC_NAME:bp3-rest-connector}")
+    private String topicName;
+
+    @Value("${camunda.bpm.client.async-response-timeout}")
+    private Long asyncResponseTimeout;
+
+    @Value("${camunda.bpm.client.subscriptions.bp3-http-json.lock-duration}")
```

Comment:
Is this name good?
```suggestion
    @Value("${camunda.bpm.client.subscriptions.bp3-rest-connector.lock-duration}")
```
Should we allow fo the lock duration to be per topic?

Reply:
Yes I need to think more about how this is all going to work. These properties are automatically loaded and applied when using the `@ExternalTaskSubscription` annotation to create the external task, and given that has to be a hardcoded topic name, then you can safely specify the topic name in the `application.properties` file properties.

But in this case whilst we can specify defaults for the `bp3-rest-connector` topic if that is used, we cannot for other topics because they could be called anything. I am sure this is not difficult to solve but I just need to find time to look at it.

Reply:
Yeah, that's what I was thinking, seems doable, but I don't know the details. Would have to move away from the @Value annotation to a method call (or set of them) after you know what the topicName is. I'm good with this it being hardcoded for now. I do think we should update the default name used though.